cli/daemon/run/infra/encorecloudtesting.go

```diff
@@ -0,0 +1,54 @@
+package infra
+
+import (
+	"encoding/base64"
+	"strconv"
+
+	"github.com/cockroachdb/errors"
+
+	"encore.dev/appruntime/exported/config"
+)
+
+// setTestEncoreCloud sets the Encore Cloud API configuration to use a local
+// Encore Cloud API server.
+//
+// It returns true if one has been configured, or false if not.
+//
+// To use it the `encore run` command must be started with the following environment variables:
+// - ENCORE_CLOUD_LOCAL_SERVER: the URL of the local Encore Cloud API server
```

Comment:
Maybe make these `ENCORECLOUD_...`? We have special code in the runtime that picks up all environment variables beginning with `ENCORE_` and these seem unrelated to the runtime itself